pages/cart.page.ts

```diff
@@ -16,4 +20,59 @@ export class CartPage {
       this.page.getByRole("button", { name: /proceed to checkout/i })
     ).toBeVisible();
   }
+
+  async expectAlreadyLoggedIn() {
+    await expect(
+      this.page.getByText(
+        "Hello Jane Doe, you are already logged in. You can proceed to checkout."
```

Comment:
What if we login as another user (not `Jane Doe`), should we creagte a new method?